### ISSUE TITLE
Fails to parse source files with specific `console.log`s

### ISSUE THREAD

**Issue by phoenisx** [stylelint/postcss-css-in-js#294]
### What steps are needed to reproduce the bug?

The source file with the following contents fails to parse:

```js
console.log(`-- foo: ${"foo"}`);
```

```js
console.log(`>>> foo--boo `, "foo");
```

```jsx
export default styled.div`
  :root {
    --h1: 44px;
    --h2: 32px;
    --h3: 24px;

    --color-red: red;
    --color-green: green;
    --color-blue: blue;
  }
`;

export const RandomComponent = () => {
  const classes = ["foo", "bar"];

  // Fails to parse or ignore the template literals below.
  return (
    <div className={`${classes[0]}`}>
      <p className={`baz ${classes[1] === "bar" ? "bizz" : "fizz"}`}>
        Hell World
      </p>
    </div>
  );
};
```

### What did you expect to happen?

Ignore such template literals, if possible.

### What actually happened?

Considers it as a CSS-in-JS template literal.

**Comment by phoenisx** [comment]
Apologies!!

I found that I didn't set up this Syntax parser properly. It's possible the error is due to that, not sure.

I tried debugging why I am not able to run this parser using `postcss-syntax` but am not able to find anything.

As per your [docs](https://github.com/stylelint/postcss-css-in-js#use-cases), the following minimal set-up should've worked right?

```js
const postcss = require("postcss");
const syntax = require("postcss-syntax");
postcss([])
  .process(source, { syntax: syntax })
  .then((result) => {});
```

If you read my [cssvar doc](https://github.com/willofindie/vscode-cssvar/blob/main/docs/customize-extension.md#example-ii), it does the same, but the parser never triggers for `jsx` files.

Can you help me out here, what can I be doing wrong here?

Also, is there a way to just `require` the parsers from this package, and run them without using `postcss-syntax`??
